integration

```diff
@@ -982,5 +982,6 @@
   "zachowj/hass-node-red",
   "zeronounours/HA-custom-component-energy-meter",
   "zigul/HomeAssistant-CEZdistribuce",
+  "zlangbert/ha-daikinone",
```

Comment:
Are you still working on this?
Other than issues/PRs opened with no response, there has not been any movement since this PR was opened?